UPGRADE_NOTES.md

```diff
@@ -22,7 +22,10 @@ brew upgrade lucky
 
 - Run `shards update`
 
-- Rename `src/server.cr` to `src/start_server.cr`. 
+- Rename `src/server.cr` to `src/start_server.cr`.
+- Edit `src/start_server.cr` by changing 
+    * `app` to `app_server` and `App` to `AppServer`.
+    * change the `puts` to output `"Listening on http://#{app_server.host}:#{app_server.port}"`
```

Comment:
This message should probably be removed actually. There is a new message that takes the place of this one and is much easier to read

Reply:
You mean the puts line? If I understand you correctly, in a future release you'll remove this puts line so may as well remove now in the upgrade notes?

Reply:
Even in the current release it is removed most of the time. It is wrapped in this if https://github.com/luckyframework/lucky_cli/blob/6796508e6a03e11bd4e1789cbe3d22dc5109d792/src/web_app_skeleton/src/start_server.cr.ecr#L9

I think it should be completely removed actually but I just need to test it out. For the upgrade notes I think we can say to remove it though.